Use up-to-date whitenoise and waitress packages (#386)
alphagov/paas-tech-docs: source/documentation/deploying_apps/deploying_django.md

4. Make sure you have all the required modules for your project installed into your virtual environment (including Django).

> 1. Generate a `requirements.txt` file if your project doesn't already have one by running `pip freeze > requirements.txt` in the root of the local folder.

Reviewer: ```suggestion
1. Generate a `requirements.txt` file if your project does not already have one by running `pip freeze > requirements.txt` in the root of the local folder.
```

>     Add the following lines to the `requirements.txt` file.

Reviewer: ```suggestion
    Add the following lines to the `requirements.txt` file:
```

        whitenoise==5.2.0  # manages static assets
        waitress==2.0.0    # a pure python WSGI server that is a replacement for gunicorn

> 4. Tell Django to use `whitenoise` to serve static files. In `settings.py` within the project folder, add this line to `MIDDLWARE`, immediately after `middleware.security.SecurityMiddleware`.

Reviewer: ```suggestion
4. Tell Django to use `whitenoise` to serve static files. In `settings.py` within the project folder, add this line to `MIDDLWARE`, immediately after `middleware.security.SecurityMiddleware`:
```

Reviewer 2: `s/MIDDLWARE/MIDDLEWARE/`

Reviewer 2: Also, should we link to the whitenoise tutorial page for this? http://whitenoise.evans.io/en/stable/django.html

Author: @0atman  Good idea. c65a864cfcc52d0321aec76c9fd0fa09120d3fb4

        'whitenoise.middleware.WhiteNoiseMiddleware',

>     To serve the files in a compressed and cachable fashion add the following line to `settings.py`.

Reviewer: ```suggestion
    To serve the files so that they can be compressed and cached, add the following line to `settings.py`:
```
